style/guide.less

```diff
@@ -0,0 +1,7 @@
+.-course-guide-table-pageids span:after {
```

Comment:
The `.less` files should never refer to classes that begin with `-`. I would just remove this file (since it's a "Crude" version anyway)